samples/third-party/gltf-loader.js

```diff
@@ -0,0 +1,354 @@
+(function() {
+    'use strict';
+
+    var GLTF = {};
+
+    var AttributeSize = {
+        5120: 1, // BYTE
+        5121: 1, // UNSIGNED_BYTE
+        5122: 2, // SHORT
+        5123: 2, // UNSIGNED_SHORT
+        5126: 4  // FLOAT
+    };
+
+    var NumberOfComponents = {
+        'SCALAR': 1,
+        'VEC2': 2,
+        'VEC3': 3,
+        'VEC4': 4,
+        'MAT2': 4,
+        'MAT3': 9,
+        'MAT4': 16
+    };
+
+    // @todo: load multiple scences
+    GLTF.Scene = function() {
+        this.vertices = [];
+        this.indices = [];
+    };
+
+    var parseGltf = function(json, onload) {
```

Comment:
Throughout use the following syntax to create functions:
```javascript
function parseGltf(json, onload) 
```
Do the same for constructor functions, but use Pascal case.